tests/test_peak_detection.py

```diff
@@ -1238,3 +1239,118 @@ def test__D4_data_causing_TwoValleysInARowError(MA202000030__2020_12_11_233215__
 
     assert np.array_equal(peak_indices, expected_peak_indices)
     assert np.array_equal(valley_indices, expected_valley_indices)
+
+
+def test__B6_data_causing_TwoValleysInARowError_with_second_peak_greater(
+    MA20123123__2020_10_13_173812__B6,
+):
+    pipeline, _ = MA20123123__2020_10_13_173812__B6
+
+    filtered_data = pipeline.get_noise_filtered_gmr()
+
+    filtered_data_vals = np.flip(filtered_data[1, :])
+    filtered_data[1, :] = filtered_data_vals
+
+    _, valley_indices = peak_detector(filtered_data, twitches_point_up=False)
```

Comment:
so it looks like `flip` worked then? is that what I'm seeing here?

if you have a big giant array of data, you can use `fmt: off` and `fmt: on` (see the `black` documentation) to keep them all on a single line.  Another choice in this case could be just to assert the 2-3 peaks of interest rather than the entire array